lib/main.dart

```diff
@@ -1,125 +1,125 @@
 import 'package:flutter/material.dart';
+import 'widget/score_card.dart';
+import 'widget/list_string_horizontal.dart';
+import 'widget/like_button.dart';
+import 'widget/movie.dart';
+import 'utils/general_constants.dart';
 
-void main() {
-  runApp(const MyApp());
-}
+void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
+  static Movie marioBros = Movie(
+    title: 'The Super Mario Bros. Movie',
+    originalTitle: 'The Super Mario Bros. Movie',
+    thumbnail: 'images/backdrop_mario_bros_movie.jpg',
+    poster: 'images/poster_mario_bros_movie.jpg',
+    overview: 'While working underground to fix a water main, Brooklyn '
+        'plumbers—and brothers—Mario and Luigi are transported down a '
+        'mysterious pipe and wander into a magical new world. But when the '
+        'brothers are separated, Mario embarks on an epic quest to find '
+        'Luigi.',
+    voteAverage: 7.8,
+    genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+    countLikes: 5,
+  );
```

Comment:
You can created a factory method that returns you this mock movie, so instead to created it on the ui side, you can have something like this:
`var movie = Movie.mockMovie();`
On movie class you are going to have something like:
`factory Movie.fromMock() => Movie(.....)`

Reply:
done